crates/bevy_render/src/render_asset.rs

```diff
@@ -161,13 +170,15 @@ impl<A: RenderAsset> RenderAssetDependency for A {
 pub struct ExtractedAssets<A: RenderAsset> {
     extracted: Vec<(AssetId<A::SourceAsset>, A::SourceAsset)>,
     removed: Vec<AssetId<A::SourceAsset>>,
+    added: Vec<AssetId<A::SourceAsset>>,
```

Comment:
Should be a HashSet over a Vec.

Reply:
Good point, will change this and `removed` as well